openedx/features/job_board/admin.py

```diff
@@ -0,0 +1,10 @@
+from django.contrib import admin
+from .models import Job
```

Comment:
please run isort in all files in which you make change

Reply:
Done